sensor_msgs/include/sensor_msgs/point_cloud2_iterator.hpp

```diff
@@ -125,6 +125,11 @@ class PointCloud2Modifier
    */
   void resize(size_t size);
 
+  /**
+   * @param size The number of T's for width and height to change the shape of the original sensor_msgs::msg::PointCloud2 by
```

Comment:
This should be separated into two params:

```suggestion
   * Updates the size and sets the width and height of the original sensors_msgs::msg::PointCloud2.
   * @param width The new width of the point cloud.
   * @param height The new height of the point cloud.
```